### ISSUE TITLE
View > Show Tab Bar Functionality

### ISSUE THREAD

**Issue by JoelProminic** [prominic/Moonshine-IDE#855]
@JustinProminic noticed a View > Show Tab Bar menu item in macOS Moonshine.  This adds a strange blank bar at the top of Moonshine.

I realized that this is a [macOS feature](https://support.apple.com/guide/mac-help/use-tabs-in-windows-mchla4695cce/mac).  I think this is supposed to be a standard interface for tabs across applications, but even in Safari the tab bar opens automatically as soon as the user opens another tab.

We could look into integrating with this feature, or trying to hide it from our menu if it doesn't make sense for us.

**Comment by JoelProminic** [comment]
I don't think we want to update tabs the way macOS expects here, so we should investigate how to disable this feature for Moonshine.   Is there something like an entitlement for this?

**Comment by rat-moonshine** [comment]
My investigation on these at here https://github.com/prominic/Moonshine-IDE/issues/856#issuecomment-866697557.

**Comment by JoelProminic** [comment]
See my clarification on https://github.com/Moonshine-IDE/Moonshine-IDE/issues/856#issuecomment-871574510 .  This is a low priority, so we don't need to spend more time on this for now.
